SourceEditor/README.md

```diff
@@ -0,0 +1,81 @@
+---
+page_type: sample
+name: "Surface Duo - SourceEditor"
+description: "Uses the dual view layout pattern and external storage capabilities to create a source-editing app for the Surface Duo."
+languages:
+- kotlin
+products:
+- surface-duo
+urlFragment: source-editor
+---
+
+# SourceEditor
+
+This sample contains a Kotlin application designed for Surface Duo. The application is an HTML editor that enables real-time rendering of formatted source code. Making use of the [dual view](https://docs.microsoft.com/dual-screen/introduction#dual-view) app pattern, users are able to edit and preview any changes simultaneously without switching windows.
+
+## Getting Started
+
+To learn how to load apps on the Surface Duo emulator, see the [documentation](https://docs.microsoft.com/dual-screen/android), and follow [the blog](https://devblogs.microsoft.com/surface-duo).
+
+## Features
+
+In contrast to the original SourceEditor [sample](https://github.com/microsoft/surface-duo-app-samples/tree/main/SourceEditor) which utilizes Microsoft's [SurfaceDuoLayout](https://docs.microsoft.com/dual-screen/android/api-reference/dualscreen-library/layouts/surfaceduo-layout) to handle dual screen devices, this project makes use of Android's [Window Manager](https://developer.android.com/jetpack/androidx/releases/window) API. This sample uses Window Manager to handle events that require consideration of a hinge or fold in the device's screen.
```

Comment:
Super minor, but here: `...makes use of Android's [Window Manager]` I think is better that we use Jetpack keyword instead of Android, so it would be: `makes use of Jetpack [Window Manager]` that is how Google refers to the Jetpack set of libraries.